### ISSUE TITLE
Cannot import sklearn.preprocessing.TargetEncoder

### ISSUE THREAD

**Issue by memeplex** [microsoft/python-type-stubs#304]
I reported https://github.com/microsoft/pylance-release/issues/5454 to pylance, but maybe it belongs here. I'm not sure whether the stubs here are distributed with pylance or not.

Notice that TargetEncoder is quite new, it was introduced in sklearn 1.3.

**Comment by Avasam** [comment]
@memeplex You are correct, the stubs found here are shipped with Pylance (but not pyright). Feel free to open a PR to add missing symbols that you need.
(note, I am not a maintainer, but I have been moving a handful of stubs here over to typeshed)
